text/0002-split-coinstake-transaction/0002-split-coinstake-transaction.md

```diff
@@ -37,21 +35,23 @@ Only the coinbase `vin[0]` has been used to carry meta-data (e.g. signaling P2SH
 Splitting the coinstake transaction into a *monetary creation* and a *coin-age consumption* transaction, has the side effect of creating small value outputs.
 Because those outputs are very unlikely to mint new blocks, node implementations are adviced to join them with a future *coin-age consumption* transaction.
 
-The minter is awarded a budget of 1 KB for the *coin-age consumption* transaction in order to adjust their UTXO table as they see fit.
-If the *coin-age consumption* transaction is greater than 1 KB, it will have to include the standard fee less 1 KB.
+The minter is currently awarded a budget of 1 KB for the coinstake transaction, and will continue to have this power.  If a *coin-age consumption* transaction is greater than 1 KB, it will have to include the standard fee less 1 KB.
 
 The diagram below illustrates the block layout with a splitted coinstake transaction.
 
 ![splitted block layout](split.png)
 
 ## Advantages
 
-* Paving the road for other improvements like multi-signature minting.
-* Limiting the power of minters to spam the blockchain.
+* Required for multi-signature minting, as described in [RFC-0003](../0003-multisig-minting/0003-multisig-minting.md)
+* While there is no known advantage for aligning the transaction structure with Bitcoin's, it should be taken as a step toward simplification, which can foster innovation.
```

Comment:
No known advantage? There is a huge advantage in not having to adapt third party tools to include the timestamp: mining pools, web wallets, ...

Reply:
This is not about the timestamp though.  Can you name a specific third party tool that would benefit from splitting the coinstake?  Because Peerchemist couldn't name one and said there wasn't anything yet, but there could be in the future.

Reply:
Great, I'll remove the 'no known advantage' and replace it with that example.